src/components/SearchByIDA/IdaEntry/index.tsx

```diff
@@ -0,0 +1,260 @@
+import React, { PureComponent } from 'react';
+
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+
+import Select, { CSSObjectWithLabel } from 'react-select';
+
+import getFetch from 'higherOrder/loadData/getFetch';
+import { format } from 'url';
+import descriptionToPath from 'utils/processDescription/descriptionToPath';
+
+import { getTrackColor, EntryColorMode } from 'utils/entry-color';
+
+import cssBinder from 'styles/cssBinder';
+import local from '../style.css';
+import { DebouncedFunc, debounce } from 'lodash-es';
+
+const css = cssBinder(local);
```

Comment:
Import EBI icon fonts

```suggestion
import fonts from 'EBI-Icon-fonts/fonts.css';
import local from '../style.css';
import { DebouncedFunc, debounce } from 'lodash-es';

const css = cssBinder(fonts, local);
```